docs/Absolute.md

```diff
@@ -0,0 +1,19 @@
+
+# Absolute
+
+Use `<Absolute />` component to control position.
+This component is an extension of the `<Box />` component.
+
+```.jsx
+<Card>
```

Comment:
Maybe make it more explicit that `Card` here should be `Relative`.  I would also think that a `Relative` component should come with the `Absolute` component. The change of position here will be relative to body, which I don't know if it's a behavior that a consumer would like seeing while wrapping it in a `Card`.